docs/README.md

```diff
@@ -124,6 +124,15 @@ Note:
 
 ## Generating a LabVIEW measurement client
 
+Note:
```

Comment:
I'm not sure this change is really worth the effort since we will probably need to undo it in the next couple of days/weeks. However, if you want to sign up for the work, I won't fight it. Once https://github.com/ni/grpc-labview/pull/387 is submitted, I think we should make a new release. Then once all of your client generator changes are submitted, I think we should make another release of this repo that depends on the latest grpc-labview release.

Reply:
I believe I can wait until the next release is made, so I will proceed to close this PR.